.gitignore

```diff
@@ -60,3 +60,8 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
```

Comment:
You ok with just putting /data here.

I don't think anything in /data should be in source control

As a convention, I like to put raw data as we receive it in /data and intermediate data steps in /work

Are you ok with this convention?  If so, we can put both /data and /work in .gitignore.